scss/bitstyles/objects/badge/_.scss

```diff
@@ -0,0 +1,26 @@
+@import './settings';
+
+.o-badge {
+  display: inline-flex;
+  align-items: center;
+  padding: $bitstyles-badge-padding-vertical $bitstyles-badge-padding-horizontal;
+  white-space: nowrap;
+  border-radius: $bitstyles-border-radius-round;
+}
+
+@each $variant in ($bitstyles-badge-variants) {
+  .o-badge--#{$variant} {
+    --button-fg: #{palette($variant, $bitstyles-badge-color)};
+    --button-bg: #{palette($variant, $bitstyles-badge-background-color)};
+    --button-fg-hover: #{palette($variant, $bitstyles-badge-background-color)};
+    --button-bg-hover: #{palette($variant, $bitstyles-badge-color)};
+
+    color: palette($variant, $bitstyles-badge-color);
+    background-color: palette($variant, $bitstyles-badge-background-color);
+  }
+}
+
+.o-badge__button {
+  margin-right: -#{spacing('s')};
+  margin-left: spacing('xs');
```

Comment:
What does the minut and brackets mean here `-#{spacing('s')}` 🤔 ?

Reply:
The `-` is there because the button needs a negative margin to pull it closer to the edge of the button (so the curves match up — see the button in the screenshots). The `#{}` part is the Sass syntax for interpolation — so it’ll render `0.6rem`, in this case.

Sadly the interpolation is required in the negative version because Sass doesn’t recognise the `-spacing` function it sees in `-spacing('xs')`